gradlew

```diff
@@ -228,11 +170,19 @@ if $cygwin ; then
     esac
 fi
 
-# Split up the JVM_OPTS And GRADLE_OPTS values into an array, following the shell quoting and substitution rules
-function splitJvmOpts() {
-    JVM_OPTS=("$@")
+# Escape application args
+save () {
```

Comment:
Did you mean `safe`?

Reply:
gradlew is generated by gradle and I didn't edit it